src/activities/assignments/AssignmentEntity.js

```diff
@@ -865,6 +943,7 @@ export class AssignmentEntity extends Entity {
 			[this.name(), assignment.name],
 			[this.instructionsEditorHtml(), assignment.instructions],
 			[this.submissionType() && String(this.submissionType().value), assignment.submissionType],
+			[this.allowableFileType() && String(this.allowableFileType().value), assignment.allowableFileType],
```

Comment:
The `equals` function here is always returning `false` because of this line. I see it locally and on the latest quad sites.

Reply:
Oh, is it `false` because the code in `activities` hasn't been merged yet? Could you double check locally if the dialog is still triggering with your activities changes.

Reply:
That's most probably the issue - I don't see that dialog when testing locally with the changes from both repos 👍 

I'm unsure how to get the `activities` repo to pick up on this repo's changes (which it depends on) - do I need to trigger a release?

Reply:
It should automatically be picked up by the activities CI.